src/app-layout/__integ__/awsui-applayout.test.ts

```diff
@@ -188,4 +188,13 @@ describe.each(['classic', 'refresh', 'refresh-toolbar'] as Theme[])('%s', theme
       expect(contentTop).toEqual(expectedOffset);
     })
   );
+
+  test(
+    'Side nav does not display a scrollbar when there is no header',
+    setupTest({ pageName: 'with-drawers-scrollable' }, async page => {
+      const navBefore = await page.getNavPosition();
+      await page.elementScrollTo(wrapper.findNavigation().toSelector(), { top: 100 });
```

Comment:
Isn't this going to fail when I merge this one: https://github.com/cloudscape-design/browser-test-tools/pull/118

Reply:
No, it shouldn't. I made it fail with the scrollbar to ensure it was working properly.